docs/conf.py

```diff
@@ -136,7 +136,7 @@ def get_tutorial_images():
 # General information about the project.
 project = 'MSS - Mission Support System'
 project_copyright = \
-    '2016-2024 by the MSS team, see AUTHORS, ' \
+    '2016-2025 by the MSS team, see AUTHORS. ' \
```

Comment:
```suggestion
    '2016-2025 by the MSS team, see AUTHORS, ' \
```

The list continues below.

Reply:
You marked this as resolved, but it isn't.

Reply:
oh bug